doc/components.md

```diff
@@ -13,12 +13,11 @@ The most commonly used implementation, `SpriteComponent`, can be created with a
 
     Sprite sprite = Sprite('player.png');
 
-    const size = 128.0;
-    var player = SpriteComponent.fromSprite(size, size, sprite); // width, height, sprite
+    final size = Vector2.all(128.0);
+    var player = SpriteComponent.fromSprite(size, sprite); // width, height, sprite
 
     // screen coordinates
-    player.x = ... // 0 by default
-    player.y = ... // 0 by default
+    player.position = ... // Vector2(0.0, 0,0) by default
```

Comment:
nit: `0.0` not `0,0`
nit: use all :)

Reply:
I think this one could stay, just so that people understand that it is x, y, but I'll change most of the rest. :)